### DISCUSSION TITLE
Generate no internet alert

### DISCUSSION THREAD

**Discussion by DavidGilWeb** [filamentphp/filament#11728] [Help]
### Package

Panel builder

### Package Version

v3.2.2

### How can we help you?

Hello, in a resource form, I would like that when an input/select is processed in live(), in case it fails because there is no internet, an error alert is displayed.
I've been trying with the function ->unsavedChangesAlerts()
But I don't see any message, it's updated to version 3.2.2 because I've seen that there were some bugs. What can I do?

**Comment by leandrocfe** [comment]
What about this?

```php
Placeholder::make('info')
    ->content(new HtmlString('<div wire:offline>There is no connection</div>'))
```

https://livewire.laravel.com/docs/offline

**Comment by DavidGilWeb** [comment]
It's good for what I want, thank you!
